### ISSUE TITLE
Ti-Multi-Standard 2.4GHz - CC2650

### ISSUE THREAD

**Issue by node4** [google/uribeacon#238]
Does anyone know if this sensor is supported? It just came out, this version at least. 
Thanks

**Comment by scottjenson** [comment]
I don't think so. Seems like a great chip and we'd love to have a version of URIBeacon running on it. I just went to their web page and the devtools are free but with any board level kit like this, it likely will take a bit of work to get the correct libraries and testing harness setup to get it working.

We'd prefer a fully V2 compliant version to be posted but if someone can just get a basic fixed ad packet working, we'll take that of course. Others can build on top of that.
